README.MD

```diff
@@ -1,14 +1,18 @@
-<h1> Minima </h1>
+<!-- markdownlint-disable no-hard-tabs no-inline-html -->
```

Comment:
Additional note: comment over here determines markdownlint's rule config, similar to `.eslintrc`. For more info, check [here](https://github.com/DavidAnson/markdownlint#configuration)